docs/mine/lotus/split-markets-miners.md

```diff
@@ -142,24 +155,32 @@ If you want to revert the changes listed above and go back to running `lotus-min
   EnableMarkets = true
 ```
 
+1. Move back the DAG store directory to the monolith miner node repository
+
+```shell
+mv ~/markets-repo-location/dagStore ~/.lotusminer/
+```
 
-3. Restart the `mining/sealing/proving` node (with the default LOTUS_MINER_PATH, which should point to your `mining/sealing/proving` node repo). Note that `lotus-miner` interacts with a given repository depending on the `LOTUS_MINER_PATH` environment variable!
+1. Restart the `mining/sealing/proving` node (with the default LOTUS_MINER_PATH, which should point to your `mining/sealing/proving` node repo). Note that `lotus-miner` interacts with a given repository depending on the `LOTUS_MINER_PATH` environment variable!
 
 ```shell
 ./lotus-miner run
 ```
 
-4. Fetch the node identity. This is necessary as you have to update your miner's peer identity on-chain, as it was changed to the identity of the markets node during the initialising of the markets service repository.
-
+1. Fetch the node identity. This is necessary as you have to update your miner's peer identity on-chain, as it was changed to the identity of the markets node during the initialising of the markets service repository.
 
 ```shell
 ./lotus-miner net id
 ```
 
-5. Update the miner's peer id on-chain with the result from the previous step.
+1. Update the miner's peer id on-chain with the result from the previous step.
 
 ```shell
 ./lotus-miner actor set-peer-id 12D3XXXXX
 ```
 
 As soon as the message is confirmed, clients will know to look for the node identity of your `mining/sealing/proving` node, which now also runs the `markets` subsystem, i.e. currently all Lotus subsystems.
+
+## DAG store troubleshooting
+
+In case anything goes wrong with the DAG store, you can always remove the `dagStore` directory and restart your miner.
```

Comment:
`If there are any problems related to the DAG store, you can safely stop the node, remove the `dagStore` directory and restart your miner. It will re-register all active deals in the dagStore directory from scratch on startup.`